File: Linear_search/JavaScript/linear_search.js
function linearSearch(arr, target) {
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] === target) {
            return i;
        }
    }
    return -1;
}

const num = [165, 1276, 16, 786, 6];
const required = 6;

const res = linearSearch(num, required);
if (res !== -1) {
    console.log(`The element ${required} was found at position ${res}`);
} else {
    console.log(`The element ${required} was not found`);
}

File: Linear_search/Csharp/linear_search.cs
using System;

class Program
{
    static int LinearSearch(int[] arr, int target)
    {
        for (int i = 0; i < arr.Length; i++)
        {
            if (arr[i] == target)
            {
                return i;
            }
        }
        return -1;
    }

    static void Main()
    {
        int[] num = { 12, 1562, 121, 1587, 74 };
        int required = 74;
        int res = LinearSearch(num, required);

        if (res != -1)
        {
            Console.WriteLine($"The element {required} was found at the position {res}");
        }
        else
        {
            Console.WriteLine($"The element {required} was not found");
        }
    }
}